scripts/utils.sh

```diff
@@ -57,19 +58,17 @@ fi
 
 trap 'exit_on_error "$0:" $?' ERR
 
-cd ${repo_root}
-
-if [[ -e ".git/rebase-apply" ]]; then
+if [[ -e "${repo_root}/.git/rebase-apply" ]]; then
   exit_on_error "!!! 'git rebase' or 'git am' in progress, aborting"
 fi
 
-git checkout -b ${temp_branch}
+git checkout -b "${temp_branch}"
 
-if [ ! -f ${repo_list} ]; then
-	touch ${repo_list}
+if [ ! -f "${repo_list}" ]; then
+	mkdir -p "${scripts_dir}" && touch "${repo_list}"
 fi
```

Comment:
Minor nit, and not something we need to tackle in this PR, but it looks like the tabbing/spacing is inconsistent throughout this script.